dataframe/src/main/groovy/com/elintegro/erf/widget/vue/TextAreaWidgetVue.groovy

```diff
@@ -20,7 +20,8 @@ class TextAreaWidgetVue extends WidgetVue {
     String getHtml(DataframeVue dataframe, Map field) {
         String fldName = dataframe.getDataVariableForVue(field)
         boolean isReadOnly = dataframe.isReadOnly(field)
-        String label = field.label
+        String mandatory = field.notNull?" *":""
```

Comment:
It would be the best if we can paint the mandatory star (*) with red. See if there is  a way to do it.